cmd/juju/application/config.go

```diff
@@ -30,76 +30,79 @@ import (
 
 const (
 	configSummary = `Gets, sets, or resets configuration for a deployed application.`
-	configDetails = `
-To view all configuration values for an application, run
+	configDetails = ``
```

Comment:
configDetails should have something. Otherwise, the help will be empty. The examples section is OK but we must put something into this variable.

Reply:
But this is just the Details bit of the Help, no? The help currently consists of the Summary, Details, Examples, See also, so even if this is empty there should still be something.